v2.11.0/language-tr.yml

```diff
@@ -0,0 +1,338 @@
+msg-trade-setup-desc-book:
+- Kitap kopyaları satar.
+- Boş yada yazılmış kitapları depolamaya ekle.
+- Sol/Sağ mouse ile miktarı ayarla.
+- 'Üst sıra: Satılacak kitap'
+- 'Alt sıra: Ücret'
+
+msg-missing-edit-villagers-perm: "&7Köylüleri düzenlemek için yetkin yok."
+msg-missing-edit-wandering-traders-perm: "&7Gezgin tüccarları düzenlemek için yetkin yok."
+msg-must-target-entity: "&7Bir varlığı hedef almalısın."
+msg-must-target-villager: "&7Bir köylüyü hedef almalısın."
+msg-target-entity-is-no-villager: "&7Hedeflenen varlık sıradan bir köylü değil."
+
+msg-villager-editor-title: "&aKöylü Editörü: &e{villagerName}"
+msg-villager-editor-description-header: "&6Köylü Editörü"
+msg-villager-editor-description:
+- 'Üst sıra: Satılacak eşya'
+- 'Alt sıra: Ücret'
+- 'Düzenlenmiş takaslar, sınırsız'
+- 'kullanılabilir ve XP vermez.'
+
+msg-button-delete-villager: "&4Sil"
+msg-button-delete-villager-lore:
+- Köylüyü siler
+msg-button-villager-inventory: "&aKöylünün envanterini gör"
+msg-button-villager-inventory-lore:
+- Köylülün envanterinin kopyasını görmeni sağlar
+msg-button-mob-ai: "&aDurumu değiştir"
+msg-button-mob-ai-lore:
+- "AI durumunu değiştirir"
+
+msg-villager-inventory-title: "Köylünün envanteri (kopya)"
+msg-set-villager-xp: "&aKöylünün XP'sini &e{xp}&a ayarla"
+msg-no-villager-trades-changed: "&aTakas değişikliği olmadı."
+msg-villager-trades-changed: "&e{changedTrades}&a takası değiştirildi."
+
+msg-list-admin-shops-header: "&9Toplamda &e{shopsCount} &9Admin marketi var: &e(Sayfa {page}/{maxPage})"
+msg-list-all-shops-header: "&9Toplamda &e{shopsCount} &9market var: &e(Sayfa {page}/{maxPage})"
+msg-list-player-shops-header: "'&e{player}&9', &e{shopsCount} &9markete sahip: &e(Sayfa {page}/{maxPage})"
+msg-list-shops-entry: "  %8ID: &e{shopId}) &8İsim: &7{shopName}&r &8Konum: &7({location})&r, %8Tür: &7{shopType}&r, %8Varlık: &7{objectType}"
+
+msg-removed-admin-shops: "&e{shopsCount} &aAdmin marketi kaldırıldı."
+msg-removed-shops-of-player: "&e{shopsCount} &a'&e{player}&a' oyuncusunun marketleri kaldırıldı."
+msg-removed-player-shops: "&e{shopsCount} &aoyuncu marketleri kaldırıldı."
+
+msg-confirm-remove-all-admin-shops: "&cBütün Admin marketlerini (&6{shopsCount}&c) geri dönüşsüz kaldırmak üzeresiniz!"
+msg-confirm-remove-all-own-shops: "&cBütün sahip olduğun marketleri (&6{shopsCount}&c) geri dönüşsüz kaldırmak üzeresiniz!"
+msg-confirm-remove-all-shops-of-player: "&6{player}&c oyuncusunun bütün marketlerini (&6{shopsCount}&c) geri dönüşsüz kaldırmak üzeresiniz!"
+msg-confirm-remove-all-player-shops: "&cBütün oyuncuların, oyuncu marketlerini (&6{shopsCount}&c) geri dönüşsüz kaldırmak üzeresiniz!!"
+
+msg-confirmation-required: "&7İşlemi onaylamak için &6/shopkeepers confirm %7yazınız!"
+msg-confirmation-expired: "&cOnaylama süresi doldu."
+msg-nothing-to-confirm: "&cŞuanda onaylanacak bir şey yok."
+
+msg-no-permission: "&cBunu yapmak için yetkin yok."
+
+msg-command-unknown: "'&e{command}&c' -> Bilinmeyen komut!"

```

Comment:
These error messages (this one and the next 11 ones as well) are all missing a `&c` at the beginning. Also, not sure why you are deviating from the default format here; maybe you can come up with something that more closely matches the default formatting of these messages. Or instead of using a `->` here, maybe use something corresponding to `&cUnknown command: '&e{command}&c'`, etc. for all these messages.